data/json/furniture_and_terrain/appliances_generators.json

```diff
@@ -1,30 +1,22 @@
 [
   {
     "type": "vehicle_part",
-    "id": "ap_active_backup_generator",
-    "name": { "str": "active backup generator" },
+    "id": "ap_portable_generator",
+    "name": { "str": "portable gasoline generator" },
     "looks_like": "f_active_backup_generator",
     "categories": [ "energy" ],
     "color": "green_white",
     "broken_color": "blue",
     "location": "structure",
     "damage_modifier": 80,
     "durability": 400,
-    "description": "A ubiquitous piece of compact machinery meant for running anything from the bathroom lights to electric stoves and other necessities when grid electricity is unavailable.",
+    "description": "A small, portable generator, one that people buy for a camp.  It just needs to be fueled up, and then turned on.",
     "//1": "BIZARRE FIELD NAMES HERE! Weird stuff happening! Epower determines how fast it is converted, power determines the ratio of diesel(mL) to battery charges output. In other words, *power* is *fuel efficiency*.",
-    "//2": "On a normal vehicle 250mL of diesel in an I4 with a truck alternator (what the active backup gen uses) should provide ~1692 charges (kW?) of power in about 24 minutes based on a test setup. That's a little less than 7W power, rounded up as I assume a larger dedicated generator is more efficient.",
     "power": "7 W",
-    "//3": "7.5 kW alternator value, minus power consumption of the engine",
-    "epower": "7300 W",
-    "//4": "This value doesn't seem to do anything for the appliance version? Let's keep it commented out for future reference.",
-    "//energy_consumption": "1000 W",
+    "epower": "3000 W",
     "m2c": 5,
-    "//5": "Actually accepts any diesel-type fuel but does NOT account for their relative fuel values! For diesel this is not a huge deal, only a few % inaccuracy, but for e.g. gas turbines this would be very bad!",
-    "fuel_type": "diesel",
-    "//6": "Appliance engine does NOT accept having these fields.",
-    "//fuel_options": [ "diesel", "biodiesel", "lamp_oil", "motor_oil", "jp8" ],
-    "//damaged_power_factor": 0.25,
-    "item": "active_backup_generator",
```

Comment:
Item should be migrated

Reply:
oh, it was not migrated? that's a problem indeed, gotta fix it right now